.pylintrc

```diff
@@ -0,0 +1,509 @@
+[MASTER]
```

Comment:
Any reason for not changing the `default_config/pylintrc` configuration in avocado-static-checks to enforce this added level of quality?  IMO that's a big part of the shared repo and the default configuration.  If there are parts of the code that should be excluded from the highest quality level checks, then a custom config can be added.